hooks/useNotification.ts

```diff
@@ -0,0 +1,9 @@
+import { useContext } from 'react';
+
+import { NotificationDispatch } from '@/providers/notificationProvider';
+
+export const useNotification = () => {
```

Comment:
```suggestion
export const useNotification = () => useContext(NotificationDispatch);
```

No need to assign to a variable

Reply:
updated